stl/inc/xmemory

```diff
@@ -471,12 +471,17 @@ struct _Is_default_allocator<allocator<_Ty>, void_t<typename allocator<_Ty>::_Fr
     : is_same<typename allocator<_Ty>::_From_primary, allocator<_Ty>>::type {};
 
 #if _HAS_CXX23
+#ifdef __cpp_lib_concepts // TRANSITION, GH-395
+template <class _Alloc, class _SizeTy, class = void>

```

Comment:
`class = void` is unnecessary for a `concept`.

Reply:
🤦